_posts/nataliele/2016-02-02-nataliele-chapter5exercise.md

```diff
@@ -0,0 +1,23 @@
+---
+layout: post
+author: nataliele
+title: "Natalie's Chapter 5 exercises"
+---
+
+Even though we havent covered lists, I feel it's impossible to finish the exercise without them so I had to consult stackexchange.
```

Comment:
I will say it's not impossible to do without lists, though, so pay attention to some of your classmates' solutions.